src/applications/form-renderer/tests/unit/utils/formConfig.unit.spec.js

```diff
@@ -179,11 +208,64 @@ describe('selectSchemas', () => {
 
     context('when militaryAddressCheckbo is false', () => {
       it('calls addressNoMilitarySchema', () => {
-        const schemas = selectSchemas(normalizedForm.chapters[3]);
+        const schemas = selectSchemas(
+          findChapterByType('digital_form_address'),
+        );
 
         expect(noMiliarySpy.calledOnce).to.eq(true);
         expect(schemas.uiSchema.address).to.not.eq(undefined);
       });
     });
   });
+
+  context('with Phone and Email Address pattern', () => {
+    const includeEmail = {
+      id: 158256,
+      chapterTitle: 'Email address included',
+      type: 'digital_form_phone_and_email',
+      pageTitle: 'Phone and email address',
+      additionalFields: {
+        includeEmail: true,
+      },
+    };
+    const phoneOnly = findChapterByType('digital_form_phone_and_email');
+
+    it('includes homePhone', () => {
+      const schemas = selectSchemas(includeEmail);
+
+      expect(schemas.schema.properties.homePhone).to.eq(
+        webComponentPatterns.phoneSchema,
+      );
+      expect(schemas.uiSchema.homePhone).to.not.eq(undefined);
+    });
+
+    it('includes mobilePhone', () => {
+      const schemas = selectSchemas(phoneOnly);
+
+      expect(schemas.schema.properties.mobilePhone).to.eq(
+        webComponentPatterns.phoneSchema,
+      );
+      expect(schemas.uiSchema.mobilePhone).to.not.eq(undefined);
+    });
+
+    context('when includeEmail is true', () => {
+      it('includes emailAddress', () => {
+        const schemas = selectSchemas(includeEmail);
+
+        expect(schemas.schema.properties.emailAddress).to.eq(
+          webComponentPatterns.emailSchema,
+        );
+        expect(schemas.uiSchema.emailAddress).to.not.eq(undefined);
+      });
+    });
+
+    context('when includeEmail is false', () => {
+      it('does not include emailAddress', () => {
+        const schemas = selectSchemas(phoneOnly);
+
+        expect(schemas.schema.properties.emailAddress).to.eq(undefined);
+        expect(schemas.uiSchema.emailAddress).to.eq(undefined);
+      });
+    });
+  });
```

Comment:
Is there a reason you set up spies for the address pattern but none of the others? Perhaps I'm missing something obvious.

Reply:
`addressSchema` is a function. The other schemas are objects. All of the uiSchemas are functions, so I could have set up spies to test that they are called. But that seemed like overkill.